src/EnergyPlus/WeatherManager.cc

```diff
@@ -144,9 +144,9 @@ namespace Weather {
 
         InitializeWeather(state, state.dataWeather->PrintEnvrnStamp);
 
-        bool anyEMSRan = false;
         // Cannot call this during sizing, because EMS will not initialize properly until after simulation kickoff
         if (!state.dataGlobal->DoingSizing && !state.dataGlobal->KickOffSimulation) {
+            bool anyEMSRan = false;
```

Comment:
[src/EnergyPlus/WeatherManager.cc:147]:(style),[variableScope],The scope of the variable 'anyEMSRan' can be reduced.